lib/push.js

```diff
@@ -38,14 +38,20 @@ module.exports = function (cfg) {
    * @param  {Object}   settings
    * @return {Function}
    */
-  function getPushClient (settings) {
+  function getPushClient(settings) {
     var sender = null;
 
-    return function push (message, options, callback) {
-      if (!message) return callback(new Error("Missing required 'message' parameter"));
-      if (!options) return callback(new Error("Missing required 'options' parameter"));
+    return function push(message, options, callback) {
+      if (!message) {
+        return callback(new Error("Missing required 'message' parameter"));
+      }
+      if (!options) {
+        return callback(new Error("Missing required 'options' parameter"));
+      }
       if (!options.broadcast) {
-        if (!options.apps) return callback(new Error("Missing required 'options.apps' parameter while 'options.broadcast' not specified" + JSON.stringify(options)));
+        if (!options.apps) {
```

Comment:
This is fine, but since we're changing it anyway, I think the two conditions can be merged. I think this will reduce the eslint warning count by 1 (no-lonely-if), and reduce the indentation level of this block.

I guess that would then be `if (!options.broadcast && !options.apps) {`

Reply:
Hi, in this case, it is not possible because the message for each is different. 

```
     if (!message) {
        return callback(new Error("Missing required 'message' parameter"));
      }
      if (!options) {
        return callback(new Error("Missing required 'options' parameter"));
      }

```